tests/introspection/tests.py

```diff
@@ -217,6 +217,21 @@ def test_get_constraints_indexes_orders(self):
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 3)
 
+    @skipUnlessDBFeature('can_introspect_foreign_keys')
+    def test_handle_space_after_foreign_key(self):
+        """
+        Test that introspection correctly handles whitespace between FOREIGN KEY
```

Comment:
We try to avoid adding `Test that...` prefixes to test documentation as it doesn't add much value. Simply mentioning that _Whitespace between FOREIGN KEY and opening parenthesis should be allowed_ should be enough here.